.typos.toml

```diff
@@ -137,8 +137,9 @@ zeroterminated = "null-terminated"
 zero-terminated = "null-terminated"
 
 [default]
-# Work around for typos inside of 8-character hashes. These show up inside of ipynb.
+extend-ignore-re = ["\"[a-f0-9]{8}\"", "_[a-f0-9]{16}"]
```

Comment:
Why is the comment after the thing it comments on? :grimacing:

Reply:
Weird, I don't even know how I made that change.